### DISCUSSION TITLE
Connect Noah2000 to the local mqtt server

### DISCUSSION THREAD

**Discussion by esanya** [mtrossbach/noah-mqtt#33] [Ideas]
Hi,

Because of a wifi network change I had to reassign noah to the new wifi network.

I have seen that there is an advanced option during the assigment process in the Shine app. 

When I clicked on the advanced option the mqtt (mqtt.growatt.com or something like this) server was shown in a disabled edit field. The edit fields are protected by password.

Is the password known to get the edit fields enabled? Is that maybe the installation code? 

Have somebody tried to 
- set the password?
- a local mqtt server?

Thx,
Regards,
Sandor

**Comment by xdatravelbug** [comment]
Hi Sandor,

What do you try to achieve?
This repository is about an add-on to be used, e.g., in HA. The above sounds like something else, not add-on related.

Best regards
